SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialOutputDevice.cpp

```diff
@@ -25,15 +26,16 @@ FSpatialOutputDevice::~FSpatialOutputDevice()
 void FSpatialOutputDevice::Serialize(const TCHAR* InData, ELogVerbosity::Type Verbosity, const class FName& Category)
 {
 	// Log category LogSpatial ignores the verbosity check.
-	if (Verbosity > FilterLevel && Category != FName("LogSpatial"))
-	{
-		return;
-	}
 
 	if (bLogToSpatial && Connection != nullptr)
 	{
 #if WITH_EDITOR
-		if (GPlayInEditorID != PIEIndex)
+		if ((Verbosity > LocalFilterLevel && Category != FName("LogSpatial")) || GPlayInEditorID != PIEIndex)
```

Comment:
This return condition can be done outside the #if #else 
```Category != FName("LogSpatial")```

Reply:
I had it outside at first but when tested it failed. It has to check the specific verbosity(local/cloud) & category

Reply:
Oh true, I missed the difference too